### PR TITLE
dovetail number 006

### PR DESCRIPTION
**Pull request by zkamvar**

### CODE REVIEW

File: _posts/2022/08/2022-08-24-dovetail-006.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+Workbench, head over to our GitHub Discussions forum: <https://github.com/carpentries/workbench/discussions>
+
+---
+
+A lot has happened since the last time I provided an update (2022-07-20), just
+not that much on The Workbench side of things. I atteneded and [presented at RStudio conference 2022](https://zkamvar.netlify.app/talk/carpentries-rstudio-2022/),
+took a short staycation to decompress (with a broken aircon, no less!),
+attended and presented at CarpentryCon 2022, and sat on hiring committees for
+our new Director of Technology. During this time, I was able to tackle some
+superficial aspects of the workbench itself.
```
Comment (tobyhodges) [inline _posts/2022/08/2022-08-24-dovetail-006.md]:
```suggestion
superficial aspects of the Workbench itself. 
```

File: _posts/2022/08/2022-08-24-dovetail-006.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+our new Director of Technology. During this time, I was able to tackle some
+superficial aspects of the workbench itself. 
+
+In this issue of The Dovetail, I wanted to reiterate the purpose of the Beta
+Phase of The Workbench (which is still on hold) because it bears repeating. We
+are going to be transitioning all of our lessons to use The Carpentries
+Workbench starting in 2023, and the Beta Phase is here to prepare the community
+for this day by doing the following:
+
+1. Creating awareness about The Workbench and [how workbench lessons differ from styles lessons](https://carpentries.github.io/workbench/transition-guide.html).
```
Comment (tobyhodges) [inline _posts/2022/08/2022-08-24-dovetail-006.md]:
```suggestion
1. Creating awareness about The Workbench and [how Workbench lessons differ from _styles_ lessons](https://carpentries.github.io/workbench/transition-guide.html).
```

File: _posts/2022/08/2022-08-24-dovetail-006.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+2. Providing a risk-free sandbox for maintainers to and instructors work with
```
Comment (tobyhodges) [inline _posts/2022/08/2022-08-24-dovetail-006.md]:
```suggestion
2. Providing a risk-free sandbox for Maintainers and Instructors to work with
```

File: _posts/2022/08/2022-08-24-dovetail-006.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+   The Workbench version of lessons before full transition.
+
+If we have the above two points, we know that we can proceed with the transition
```
Comment (tobyhodges) [inline _posts/2022/08/2022-08-24-dovetail-006.md]:
```suggestion
If we have succeeded in the above two points, we know that we can proceed with the transition
```

File: _posts/2022/08/2022-08-24-dovetail-006.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+of our lessons in 2023. However, The Workbench Beta Phase is designed to do
+more than simply making the community aware of the changes; we want to make
```
Comment (tobyhodges) [inline _posts/2022/08/2022-08-24-dovetail-006.md]:
```suggestion
more than simply make the community aware of the changes; we want to make 
```

File: _posts/2022/08/2022-08-24-dovetail-006.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+2. Providing a risk-free sandbox for maintainers to and instructors work with
+   The Workbench version of lessons before full transition.
+
+If we have the above two points, we know that we can proceed with the transition
+of our lessons in 2023. However, The Workbench Beta Phase is designed to do
+more than simply making the community aware of the changes; we want to make 
+sure The Workbench is _thriving_ in the hands of the community. That is, we want
+to: 
+
+3. Make sure that the tool actually meets the needs and wants of the community.
```
Comment (tobyhodges) [inline _posts/2022/08/2022-08-24-dovetail-006.md]:
```suggestion
3. Ensure that the tool actually meets the needs and wants of the community. 
```

Avoid repeating "make sure" in quick succession.

File: _posts/2022/08/2022-08-24-dovetail-006.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+   The Workbench version of lessons before full transition.
+
+If we have the above two points, we know that we can proceed with the transition
+of our lessons in 2023. However, The Workbench Beta Phase is designed to do
+more than simply making the community aware of the changes; we want to make 
+sure The Workbench is _thriving_ in the hands of the community. That is, we want
+to: 
+
+3. Make sure that the tool actually meets the needs and wants of the community. 
+4. Create champions future contributors to The Workbench.
```
Comment (tobyhodges) [inline _posts/2022/08/2022-08-24-dovetail-006.md]: Not sure what you are trying to say here? Maybe "Inspire future contributors to The Workbench"?

Comment (tobyhodges) [reply _posts/2022/08/2022-08-24-dovetail-006.md]: How realistic is this as a goal for the Beta? I imagine future contributors would need to become familiar with the source code of the Workbench packages, and I don't think that is built into the current plan for the Beta?

File: _posts/2022/08/2022-08-24-dovetail-006.md [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+
+When I piloted the [Alpha Test of The Workbench]({{ site.url
+}}/blog/2021/07/infrastructure-testing), my goal was to simply make sure that
```
Comment (tobyhodges) [inline _posts/2022/08/2022-08-24-dovetail-006.md]:
```suggestion
}}/blog/2021/07/infrastructure-testing), my goal was to make sure that
```

Don't undersell yourself - it didn't look simple to me!

File: _posts/2022/08/2022-08-24-dovetail-006.md [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+Workbench](https://carpentries.github.io/sandpaper-docs/) has been continuously
+hosted for the last year and a half, since 2021-02-09), and I am ready for
+early adopters to try it out and really tell me how they _feel_ about The
+Workbench.
+
+This can only be really acheived with directed and specific feedback, which is
+why I've given the [Beta Phase a fairly rigid structure in three stages]({{ site.url }}/blog/2022/05/workbench-beta/#beta-logistics).
+During each stage, I will be asking Maintainers and Instructors to choose from
+and try out different, specific, tasks and give feedback in GitHub Issue on The
+Workbench repository. For an extra level of detail, I am hoping that I can find
```
Comment (tobyhodges) [inline _posts/2022/08/2022-08-24-dovetail-006.md]:
```suggestion
During each stage, I will be asking Maintainers and Instructors to
try out different, specific, tasks and give feedback in GitHub Issues on 
[The Workbench repository](https://github.com/carpentries/workbench).
For an extra level of detail, I am hoping that I can find
```

File: _posts/2022/08/2022-08-24-dovetail-006.md [hunk 10]
```diff
@@ -0,0 +1,179 @@
...
+some maintainers or instructors to provide [Friction Logs](https://github.com/carpentries/workbench/discussions/2),
```
Comment (tobyhodges) [inline _posts/2022/08/2022-08-24-dovetail-006.md]:
```suggestion
some Maintainers or Instructors to provide [Friction Logs](https://github.com/carpentries/workbench/discussions/2),
```

File: _posts/2022/08/2022-08-24-dovetail-006.md [hunk 11]
```diff
@@ -0,0 +1,179 @@
...
+which will give me a step-by-step view into a specific process so that the
+exact cause of the friction can be identified and smoothed out.
```
Comment (tobyhodges) [inline _posts/2022/08/2022-08-24-dovetail-006.md]:
```suggestion
exact causes of friction can be identified and smoothed out.
```

**Review by tobyhodges — CHANGES_REQUESTED** [review]
Sets expectations and gives testers a bit more information about what they will be asked to do in the Beta 👍  I have made a few corrections and suggestions.

File: _posts/2022/08/2022-08-24-dovetail-006.md (hunk 7 above)
Comment (zkamvar) [reply _posts/2022/08/2022-08-24-dovetail-006.md]: This is a good point. Its less of a goal and more of a side-effect from transparently fixing issues and crediting community members in places where they were having trouble.  I've updated the wording to reflect this aspiration.

```suggestion
4. Create champions of, and inspire future contributors to The Workbench.
```

**Review by tobyhodges — APPROVED** [review]

File: _posts/2022/08/2022-08-24-dovetail-006.md [hunk 12]
```diff
@@ -0,0 +1,180 @@
...
+title: "The Dovetail #6: Updates from The Carpentries Workbench"
+date: 2022-08-24
+time: "09:00:00"
+tags: ["Lesson Infrastructure", "Community", "Carpentries Workbench", "Beta", "Dovetail"]
+---
+
+This is the sixth post in a series that we are calling "The Dovetail."
+In this series, we aim to keep members of The Carpentries community abreast of
+the current news for [The Carpentries Workbench](https://carpentries.github.io/workbench). 
+
```
Comment (kmomar) [inline _posts/2022/08/2022-08-24-dovetail-006.md:15]: We should add a sentence and link around this area to the rest of the posts if people are interested

File: _posts/2022/08/2022-08-24-dovetail-006.md [hunk 13]
```diff
@@ -0,0 +1,180 @@
...
+If you are interested in participating in discussions around The Carpentries
+Workbench, head over to our GitHub Discussions forum: <https://github.com/carpentries/workbench/discussions>
+
+---
+
+A lot has happened since the last time I provided an update (2022-07-20), just
+not that much on The Workbench side of things. I atteneded and [presented at RStudio conference 2022](https://zkamvar.netlify.app/talk/carpentries-rstudio-2022/),
```
Comment (kmomar) [inline _posts/2022/08/2022-08-24-dovetail-006.md]:
```suggestion
not that much on The Workbench side of things. I attended and [presented at the RStudio conference 2022](https://zkamvar.netlify.app/talk/carpentries-rstudio-2022/),
```

File: _posts/2022/08/2022-08-24-dovetail-006.md [hunk 14]
```diff
@@ -0,0 +1,180 @@
...
+took a short staycation to decompress (with a broken aircon, no less!),
```
Comment (kmomar) [inline _posts/2022/08/2022-08-24-dovetail-006.md]:
```suggestion
took a short "staycation" to decompress (with a broken aircon, no less!),
```

**Review by kmomar — CHANGES_REQUESTED** [review]
Just some short copy edits! Great work @zkamvar

File: _posts/2022/08/2022-08-24-dovetail-006.md (hunk 12 above)
Comment (zkamvar) [reply _posts/2022/08/2022-08-24-dovetail-006.md:15]: I agree. Do you want to add this before you merge?

File: _posts/2022/08/2022-08-24-dovetail-006.md [hunk 15]
```diff
@@ -0,0 +1,180 @@
...
+teaser: "Hello Again"
+title: "The Dovetail #6: Updates from The Carpentries Workbench"
+date: 2022-08-24
+time: "09:00:00"
+tags: ["Lesson Infrastructure", "Community", "Carpentries Workbench", "Beta", "Dovetail"]
+---
+
+This is the sixth post in a series that we are calling "The Dovetail."
+In this series, we aim to keep members of The Carpentries community abreast of
+the current news for [The Carpentries Workbench](https://carpentries.github.io/workbench).
```
Comment (tobyhodges) [inline _posts/2022/08/2022-08-24-dovetail-006.md]:
```suggestion
This is the sixth post in [a series that we are calling "The Dovetail",
about the transition to The Carpentries Workbench](https://carpentries.org/posts-by-tags/#blog-tag-dovetail).
In this series, we aim to keep members of The Carpentries community abreast of
the current news about [the Workbench](https://carpentries.github.io/workbench). 
```

File: _posts/2022/08/2022-08-24-dovetail-006.md (hunk 12 above)
Comment (zkamvar) [reply _posts/2022/08/2022-08-24-dovetail-006.md:15]: It looks like toby made the changes. Feel free to merge

**Review by kmomar — APPROVED** [review]

**Pull request merged**